ddtrace/internal/datadog/profiling/dd_wrapper/src/code_provenance.cpp

```diff
@@ -83,9 +83,6 @@ CodeProvenance::add_filename(std::string_view filename)
 
     const Package* package = it->second.get();
     if (package) {
-        if (packages_to_files.find(package) == packages_to_files.end()) {
```

Comment:
Ah nice, avoiding unneeded lookups :) Not being familiar with C++, I had to check the docs, but I see that the map will indeed create a default-constructed value for the key if it's not found: https://en.cppreference.com/w/cpp/container/unordered_map/operator_at TIL!